test/parallel/test-https-server-keep-alive-timeout.js

```diff
@@ -31,20 +31,18 @@ function run() {
 
 test(function serverKeepAliveTimeoutWithPipeline(cb) {
   let socket;
-  let destroyedSockets = 0;
   let timeoutCount = 0;
   let requestCount = 0;
   process.on('exit', function() {
     assert.strictEqual(timeoutCount, 1);
     assert.strictEqual(requestCount, 3);
```

Comment:
I agree that since this PR is focused on robustness, it will be "offtopic" to remove, but now it "cries out" to remove this in favor of a `mustCall(x,3)` in line (new file) 37, and corresponding 63 & 66.